config/config.yaml

```diff
@@ -0,0 +1,8 @@
+# config.yaml
+directories:
+  image_dir: /path/to/images
+  output_dir: /path/to/output
+
+arguments:
+  include_tags: ""
```

Comment:
put in the tags we use

Reply:
Let's update the ImageTagger, decoupling the work to create/save CLIP embeddings from the work to match against tags.

Tags will be high-level, thematic descriptors to help filter out unsuitable images like screenshots/art/aerial imagery and to select for the typical first person perspectives of indoor/outdoor scenes. Ideal scenes will have a few well-separated objects, easily distinguished by appearance or spatial relation, convex objects occupying large parts of the image plane.

Generating embeddings in one pass for all the images, we can use zero-shot matching with text embeddings to filter over additional stages. For example, tag images by characteristic scale length "1 inch scale", "1 foot scale", "10 foot scale", "100 foot scale"